common/src/androidTest/kotlin/DatabaseTest.kt

```diff
@@ -0,0 +1,83 @@
+import com.cuhacking.atlas.db.*
+import com.cuhacking.atlas.db.Database.Companion.Schema
+import com.squareup.sqldelight.db.SqlDriver
+import com.squareup.sqldelight.sqlite.driver.JdbcSqliteDriver
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import kotlin.test.assertTrue
+
+lateinit var driver: SqlDriver
+lateinit var queries: FeatureQueries
```

Comment:
These can be moved into the `DatabaseTest` class. The `queries` variable can be removed. We can/should just get the queries from the database at any time.

Reply:
I believe there isn't a query that allows the insertion of multiple queries at once, so a scope function would be the best solution like you mentioned!